test/integration/kubernetes_deploy_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class KubernetesDeployTest < KubernetesDeploy::IntegrationTest
+  def test_basic
+    runner = KubernetesDeploy::Runner.new(
+      namespace: @namespace,
+      current_sha: "abcabcabc",
+      context: "minikube",
```

Comment:
Installing `minikube` automatically creates the `minikube` context, so this value won't change in future. Meaning that we'll always be using `minikube` context in test environment.
I'm totally 👍 for moving it into a const.